### DISCUSSION TITLE
Is it possible make the "block-menu-dom" in plugin-block to support a react component?

### DISCUSSION THREAD

**Discussion by 4-1-1** [Milkdown/milkdown#708] [Q&A] [answered]
To make some awesome tool  in the menu . like table selector 
<img width="535" alt="image" src="https://user-images.githubusercontent.com/4401981/188247883-dc9584ca-27b9-411b-9165-21c914da1873.png">

**Comment by Saul-Mirone** [comment, marked as answer]
Maybe it's possible by react portal. The plugin block needs to add some new options. I'll give it a try when I have time.
